src/dials/command_line/stills_process.py

```diff
@@ -402,7 +402,9 @@ def run(self, args=None):
                 all_paths.extend(params.input.glob)
             globbed = []
             for p in all_paths:
-                globbed.extend(glob.glob(p))
+                g = glob.glob(p)
+                assert g, f"Unhandled path or option: {p}"
```

Comment:
Would this be better as a sys.exit, which is more readable than a stack trace?
```suggestion
                if not g:
                    sys.exit(f"Error: Unhandled path or option: {p}")
```